setup.py

```diff
@@ -25,6 +25,7 @@
     "numpy>=1.16",
     "xarray>=0.14.1",
     "scipy>=1.2",
+    "numba",
```

Comment:
Should we peg a version?

Reply:
I thought of it, but AFAIK,  the minimum version with `jit` and `vectorize` (with the unctionalities we need) has numpy 1.16 as a dependency. So, pinning numpy implicitly pins numba. If you think explicit pinning is  better, let's do it, I don't these things...

Reply:
That should be fine then. It might be an issue for the conda package but I'll deal with it if it comes up.